WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsViewController.swift

```diff
@@ -119,7 +119,7 @@ struct InPersonPaymentsView: View {
 struct InPersonPaymentsView_Previews: PreviewProvider {
     static var previews: some View {
         NavigationView {
-            InPersonPaymentsView(viewModel: InPersonPaymentsViewModel(fixedState: .completed(plugin: .stripeOnly)))
+            InPersonPaymentsView(viewModel: CardPresentPaymentOnboardingViewModel(fixedState: .completed(plugin: .stripeOnly)))
```

Comment:
Hmm we should also rename the view, shouldn't we. I'll do that.

Reply:
My bad 🤦🏻‍♀️ thanks for updating the view & view controller!